thinc/extra/wrappers.py

```diff
@@ -119,8 +123,7 @@ def backward_pytorch(dy_data, sgd=None):
                 self._optimizer.step()
                 self._optimizer.zero_grad()
             return dX
-        # TODO: We should return also h_n e.g. for seq2seq models
-        return self.ops.asarray(out.data), backward_pytorch
+        return (self.ops.asarray(out.data), h_n) , backward_pytorch_rnn
```

Comment:
Some pending issues: 
- h_n should not be PyTorch specific when send outside this function
- For LSTMs h_n will contain a tuple with h_n and c_n (the cell state for t=seq_len)